example/glue/ScavengerBackOutScanner.hpp

```diff
@@ -70,6 +70,13 @@ class MM_ScavengerBackOutScanner : public MM_Base
 			}
 			objectEntry = (ObjectEntry *)hashTableNextDo(&state);
 		}
+		OMR_VMThread *walkThread;

```

Comment:
Again, this would require significant refactoring of Java implementation of root/backout scanners. These use MM_RootScanner, which extracts saved objects in doVMThreadSlot(). I'm not a fan of MM_RootScanner, but it would be difficult to refactor it to accommodate this.